src/lib/functions/server.js

```diff
@@ -105,6 +106,11 @@ const createHandler = function ({ functionsRegistry }) {
       rawQuery,
     }
 
+    if (config.authlify && config.authlify.authlifyTokenId != null) {
```

Comment:
follows https://github.com/netlify/proxy/pull/719 by injecting the token whenever there's an authlify token ID, and not only when `--graph` is passed.